flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/IntermediateResultPartitionTest.java

```diff
@@ -251,11 +279,24 @@ public static ExecutionGraph createExecutionGraph(
             v2.setMaxParallelism(consumerMaxParallelism);
         }
 
-        v2.connectNewDataSetAsInput(v1, distributionPattern, ResultPartitionType.BLOCKING);
+        final JobVertex v3 = new JobVertex("v3");
```

Comment:
To double confirm, is this change helps to verify that the previous test can still pass with no exception with multi consumer job vertices?

Reply:
Yes, exactly.